README.md

```diff
@@ -1,32 +1,205 @@
+DETAIL:  Key (id)=(1) already exists.
+SQL state: 23505
+```
+
+
+We can confirm a new row was inserted by executing a `SELECT` statement:
+
+```sql
+SELECT *
+FROM pet;
+```
+
+![Select statement](https://curriculum-content.s3.amazonaws.com/6036/sql-insert-statement/select1row.png)
+
+
+Let's insert a few more rows into the `pet` table.
+
+```sql
+INSERT INTO pet (id, name, species, breed, age) 
+VALUES (2, 'Hana', 'cat', 'Tabby', 1);
+
+-- Use two single quotes '' to escape a single quote character in the name
+INSERT INTO pet (id, name, species, breed, age) 
+VALUES (3, 'Lil'' Bub', 'cat', 'American Shorthair', 5);
+
+INSERT INTO pet (id, name, species, breed, age) 
+VALUES (4, 'Snuggles', 'dog', 'Bichon', 5);
+
+--unknown age
+INSERT INTO pet (id, name, species, breed)
+VALUES (5, 'Honey', 'dog', 'Cavachon');
+
+-- unknown breed
+INSERT INTO pet (id, name, species, age)
+VALUES (6, 'Herbie', 'fish', 4);
+```
+
+- Lil' Bub the cat requires two single quotes between "Lil" and "Bub" to escape the single quote character.
+- Honey the dog is not given a value for age.  The value `null` will be stored in that column.
+- Herbie the fish is not given a value for breed.  The value `null` will be stored in that column.
+- An SQL comment line starts with two dashes -- followed by a space.
+
+The query tool can execute multiple SQL statements.
+
+1. Enter all 5 INSERT statements into the query panel.
+2. Press the Execute button to insert the 5 rows.
+3. The message should confirm the statement was successful.
+
+![insert 5 rows](https://curriculum-content.s3.amazonaws.com/6036/sql-insert-statement/insert5rows.png)
+
+
+
+Query the `pet` table to confirm it now contains 6 rows:
+
+![select all rows](https://curriculum-content.s3.amazonaws.com/6036/sql-insert-statement/select6rows.png
+)
+
+
+The INSERT statement does not require the
+comma-separated list of column names
+if the number of values matches the number of columns.
+For example, since the `pet` table contains 5 columns,
+we could have written:
+
+```sql
+INSERT INTO pet 
+VALUES (2, 'Hana', 'cat', 'Tabby', 1);
+```
+
+However, it is a good idea to explicitly list the column
+names to ensure we provide the values in the correct order.
+Also, we must provide the column names if a value
+is not provided for one or more columns as was the case
+for Honey the dog (unknown age) and Herbie the fish (unknown breed).
+
+
+## UPDATE
+
+The SQL `UPDATE` statement changes the column value(s) for one or more rows.
+
+The general syntax is:
+
+```sql
+UPDATE [table_name] 
+SET column_name = value
+WHERE [row selection predicate]
+```
+
+For example:
+
+```SQL
+UPDATE pet
+SET age = 7
+WHERE id = 5;
+```
+
+NOTE: It is very important to ensure the `WHERE` clause is correct, otherwise we might
+update the wrong row!    It is usually a good idea to first write a query using the `SELECT`
+statement with the **before** executing an update to see which rows will be affected:
```

Comment:
Perhaps remove the words "with the" here

Reply:
fixed typo.